### PR TITLE
Added new layout in CSS and adpat front to new API

### PR DESCRIPTION
**Pull request by ar4s**
Added visualization rear of chassis.

### CODE REVIEW

File: src/ralph/ui/static/partials/rack/rack_view.html [hunk 1]
```diff
@@ -97,7 +97,7 @@
         </table>
     </div>
     <div class="racks">
-        <rack info="rack.info" devices="rack.front" side="front"></rack>
-        <rack info="rack.info" devices="rack.back" pdus="rack.pdus" side="back"></rack>
+        <rack info="rack.info" devices="rack.devices" side="front" ng-init=""></rack>
```
Comment (xliiv) [inline src/ralph/ui/static/partials/rack/rack_view.html]: rm it, please

File: src/ralph/ui/static/partials/rack/rack.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
             <pdu-item pdu="item" ng-repeat="item in pdus|filter:{orientation:'left'}" style="order: {{ $index }}"></pdu-item>
         </div>
         <div class="devices">
-            <device ng-repeat="item in devices"></device>
+            <device item="item" side="side" ng-repeat="item in devices"></device>
```
Comment (quamilek) [inline src/ralph/ui/static/partials/rack/rack.html]: Rename `item` to `rack_item`, rename `devices` to `rack_items`.

File: src/ralph/ui/static/partials/rack/rack_view.html [hunk 3]
```diff
@@ -97,7 +97,7 @@
         </table>
     </div>
     <div class="racks">
-        <rack info="rack.info" devices="rack.front" side="front"></rack>
-        <rack info="rack.info" devices="rack.back" pdus="rack.pdus" side="back"></rack>
+        <rack info="rack.info" devices="rack.devices" side="front"></rack>
+        <rack info="rack.info" devices="rack.devices" pdus="rack.pdus" side="back"></rack>
```
Comment (quamilek) [inline src/ralph/ui/static/partials/rack/rack_view.html]: ditto

File: src/ralph/ui/static/ui/js/app/rack/directives.js [hunk 4]
```diff
@@ -17,15 +17,22 @@ angular
     .directive('device', function () {
         return {
             restrict: 'E',
+            scope: {
+                item: '=',
```
Comment (quamilek) [inline src/ralph/ui/static/ui/js/app/rack/directives.js]: ditto

**Comment by xliiv** [comment]
:ok:

**Comment by quamilek** [comment]
:ok:

**Pull request merged**